translatepy/translators/yandex.py

```diff
@@ -116,65 +71,21 @@ def translate(self, text, destination_language, source_language="auto") -> Union
             if source_language is None or str(source_language) == "auto":
                 source_language = self.language(text)
                 if source_language is None:
+                    # TODO: Raise exception
```

Comment:
Don't think we should raise an exception as everything is made to be as simple as possible and server-ready: if there is an error or anything which happened during the process: return None

Reply:
Hmm, but this makes it very difficult to work with the code, it is not clear where exactly the error occurs, I had to replace:
```
try:
    ...
except Exception:
    pass
```
with:
```
try:
    ...
except ImportError:
    pass
```
to identify the problems